plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestMemSqlDistributedQueries.java

```diff
@@ -120,11 +120,6 @@ protected TestTable createTableWithDefaultColumns()
             return Optional.empty();
         }
 
-        if (typeName.contains("char(1)")) {
```

Comment:
With the TODO removals being scattered across commits, it's harder to see how many of them remained.
Also, the code in all these connectors looks very repetitive, so I think you can safely squash up the commit adding tests and commits fixing the problem across connectors, without any downside on the reviewability.